package.json

```diff
@@ -303,7 +303,7 @@
         {
           "command": "mssql.createAzureFunction",
           "when": "view == objectExplorer && viewItem == Table",
-          "group": "MS_SQL@6"
+          "group": "zAzure_Function@1"
```

Comment:
what's the `z`? Is that part of the group name, or does that mean something else?

Reply:
Yes. VS Code orders groups alphabetically so putting z on the group name makes it so it's at the end (which this should be).

AFAIK there isn't any other way to force the ordering, not sure why they don't have an "order" field or something 🤷

Reply:
hah ok 😄

Reply:
Another common pattern is to put numbers, e.g. 

1_MS_SQL
2_Azure_Function

Reply:
https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups

Reply:
The group order is based on the alphabetical order of the group, so this ensures that it's below the second group.